v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/datastream/ChangeEventTypeConvertor.java

```diff
@@ -78,7 +78,7 @@ public static Long toLong(JsonNode changeEvent, String key, boolean requiredFiel
     try {
       JsonNode node = changeEvent.get(key);
       if (node.isTextual()) {
-        return Double.valueOf(node.asText()).longValue();
+        return Long.valueOf(node.asText());
       }
       return node.asLong();
```

Comment:
Is this line covered in unit test (i.e node.isTextual is false)? If not, please add tests to cover this branch as well.

Reply:
All branches are covered, PTAL - 

https://screenshot.googleplex.com/85pADZkWq3ErpCQ